### PR TITLE
[ru] Localization of concepts/workloads/controllers/cron-jobs.md

### PR DESCRIPTION
**Pull request by yungserge**
This contributes to issue #37044.
Previewed the changes locally.

This one was tough to translate, I used my best judgement, and tried to refer to already translated documents to see what words are used there for the concepts. However, some terms are pretty abstract so I am looking forward to the feedback.
For example, I did not translate the word CronJob but instead kept it and applied Russian conjugation to it assuming it's a masculine noun. It sounds a little awkward so sometimes I used the expression "Задание CronJob". If there is a preferred way to handle this or some other reoccurring issue, let me know, I can go ahead and edit every instance. 
Thank you!

/language ru

### CODE REVIEW

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+сделанная позже, не будет полезной: лучше дождаться следующего запуска по расписанию.
+
+Задания, которые не успели выполниться в установленный срок, Kubernetes рассматривает как неудачные.
+Если не указать `startingDeadlineSeconds` для CronJob, то задания не будут иметь дедлайн.
+
+Если поле `.spec.startingDeadlineSeconds` установлено (не является нулевым), контроллер CronJob
+измеряет время между моментом, когда задание должно быть создано, и реальным временем.
+Если разница превышает этот предел, он пропустит  операцию.
+
+Например, если установлено значение `200`, это позволяет создать задание на 200 секунд после фактического расписания.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:112]:
```suggestion
Например, если установлено значение `200`, задание может быть создано в период до 200 секунд позже фактического расписания.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+### Правила параллелизма
+
+Поле `.spec.concurrencyPolicy` также является необязательным.
+Оно определяет, как обращаться с одновременным выполнением заданий, созданных этим CronJob.
+В этой спецификации может быть указана только одна из следующих директив параллелизма:
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:118]:
```suggestion
В этой спецификации может быть указана только одна из следующих политик параллелизма:
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+Задания, которые не успели выполниться в установленный срок, Kubernetes рассматривает как неудачные.
+Если не указать `startingDeadlineSeconds` для CronJob, то задания не будут иметь дедлайн.
+
+Если поле `.spec.startingDeadlineSeconds` установлено (не является нулевым), контроллер CronJob
+измеряет время между моментом, когда задание должно быть создано, и реальным временем.
+Если разница превышает этот предел, он пропустит  операцию.
+
+Например, если установлено значение `200`, это позволяет создать задание на 200 секунд после фактического расписания.
+
+### Правила параллелизма
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:114]:
```suggestion
### Политика параллелизма (concurrency policy)
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+* `Allow` (по умолчанию): CronJob разрешает одновременное выполнение заданий.
+* `Forbid`: CronJob не разрешает одновременное выполнение заданий; если пришло время для выполнения нового задания,
+  а предыдущее еще не завершилось, CronJob пропускает новое задание.
+* `Replace`: Если пришло время для выполнения нового задания, а предыдущее еще не завершилось,
+  CronJob заменит текущее задание на новое.
+
+Заметьте, что принцип параллелизма применяется только к заданиям, созданным одним и тем же Cronjob'ом.
+Если есть несколько CronJob'ов, их соответствующие задания всегда могут выполняться одновременно.
+
+### Остановка графика (Schedule suspension)
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:129]:
```suggestion
### Остановка выполнения заданий по расписанию
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+Можно приостановить выполнение CronJob заданий, присвоив необязательному полю `.spec.suspend` значение true.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:131]:
```suggestion
Можно приостановить выполнение CronJob-заданий, присвоив необязательному полю `.spec.suspend` значение true.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+По умолчанию это поле имеет значение false.
+
+Эта настройка _не_ влияет на задания, которые уже были запущены CronJob'ом.
+
+Если установить это поле как true, все последующие выполнения будут приостановлены
+(они остаются запланированными, но CronJob контроллер не запускает задания для выполнения),
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:137]:
```suggestion
(они остаются запланированными, но CronJob-контроллер не запускает задания для выполнения),
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+пока вы не отмените приостановку CronJob'a.
+
+{{< caution >}}
+Работы, приостановленные во время запланированного выполнения, считаются пропущенными.
+Когда `.spec.suspend` изменяется с `true` на `false` для существующего CronJob'a без
+[дэдлайна задержки начала задания](#starting-deadline), пропущенные задания планируются немедленно.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:143]:
```suggestion
[дедлайна задержки начала задания](#starting-deadline), пропущенные задания планируются немедленно.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+{{< /caution >}}
+
+### Лимиты на историю заданий
+
+Поля `.spec.successfulJobsHistoryLimit` и `.spec.failedJobsHistoryLimit` являются опциональными.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:148]: Just for the sake of consistency in this document.
```suggestion
Поля `.spec.successfulJobsHistoryLimit` и `.spec.failedJobsHistoryLimit` являются необязательными.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+В этих полях указывается, сколько завершенных и неудачных заданий должно быть сохранено.
+По умолчанию они установлены на 3 и 1 соответственно.  Установка предела в `0` соответствует сохранению
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:150]:
```suggestion
По умолчанию они установлены на 3 и 1 соответственно. Установка предела в `0` будет означать,
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+ни одного из заданий соответствующего типа после их завершения.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:151]:
```suggestion
что никакие задания соответствующего типа не сохранятся после их завершения.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+{{< caution >}}
+Работы, приостановленные во время запланированного выполнения, считаются пропущенными.
+Когда `.spec.suspend` изменяется с `true` на `false` для существующего CronJob'a без
+[дэдлайна задержки начала задания](#starting-deadline), пропущенные задания планируются немедленно.
+{{< /caution >}}
+
+### Лимиты на историю заданий
+
+Поля `.spec.successfulJobsHistoryLimit` и `.spec.failedJobsHistoryLimit` являются опциональными.
+В этих полях указывается, сколько завершенных и неудачных заданий должно быть сохранено.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:149]:
```suggestion
В этих полях указывается, сколько завершенных и неудачных заданий должно быть сохранено в истории.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+
+### Часовые пояса
+
+{{< feature-state for_k8s_version="v1.27" state="stable" >}}
+
+Для CronJob'a, в которых не указан часовой пояс, {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+интерпретирует расписания относительно своего локального часового пояса.
+
+Можно указать часовой пояс для CronJob'a, присвоив `.spec.timeZone` имя действительного
+[часового пояса](https://ru.wikipedia.org/wiki/Список_часовых_поясов_по_странам)
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:163]: TZ identifiers for the time zones are missing in Russian Wikipedia. Therefore, we'd better use the original link here.
```suggestion
[часового пояса](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+Например, установка `.spec.timeZone: "Etc/UTC"` инструктирует Kubernetes интерпретировать расписание
+относительно всемирного координированного времени.
+
+База данных часовых поясов из стандартной библиотеки Go включена в бинарные файлы и используется в качестве запасного варианта на случай, если внешняя база данных недоступна в системе.
+
+## Ограничения CronJob {#cron-job-limitations}
+
+### Неподдерживаемая TimeZone спецификация
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:171]:
```suggestion
### Неподдерживаемая спецификация TimeZone
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+
+Установка часового пояса с помощью переменных `CRON_TZ` или `TZ` в `.spec.schedule`
+**не поддерживается официально** (и никогда не поддерживалась).
+
+Начиная с Kubernetes 1.29, если попытаться задать расписание,
+включающее спецификацию часового пояса `TZ` или `CRON_TZ`,
+Kubernetes не сможет создать ресурс, выдав ошибку валидации.
+Обновления CronJobs, уже использующих `TZ` или `CRON_TZ`, будут продолжать выдавать
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:179]:
```suggestion
Обновления для CronJobs, уже использующих `TZ` или `CRON_TZ`, будут продолжать выдавать
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 15]
```diff
@@ -0,0 +1,233 @@
...
+модификации. Задания (и их поды), которые уже были запущены, продолжат выполняться без изменений.
+То есть CronJob _не_ обновляет существующие задания, даже если они остаются запущенными.
+
+### Создание заданий
+
+CronJob создает объект задания (Job) примерно один раз за время выполнения своего расписания.
+Расписание является приблизительным, поскольку есть определенные обстоятельства,
+при которых могут быть созданы два задания или не создано ни одного.
+Kubernetes старается избегать таких ситуаций, но не может полностью предотвратить их. Поэтому
+задания, которые вы определяете, должны быть _эдемпотентными_.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:196]:
```suggestion
задания, которые вы определяете, должны быть _идемпотентными_.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 16]
```diff
@@ -0,0 +1,233 @@
...
+
+Если поле `startingDeadlineSeconds` задано большим значением или не задано (по умолчанию),
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:198]:
```suggestion
Если в поле `startingDeadlineSeconds` задано большое значение или оно не определено (по умолчанию),
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 17]
```diff
@@ -0,0 +1,233 @@
...
+и,если `concurrencyPolicy` имеет значение `Allow`, задания всегда будут выполняться по крайней мере один раз.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:199]:
```suggestion
а также при условии, что `concurrencyPolicy` имеет значение `Allow`, задания всегда будут выполняться по крайней мере один раз.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 18]
```diff
@@ -0,0 +1,233 @@
...
+
+{{< caution >}}
+Если `startingDeadlineSeconds` имеет значение меньше 10 секунд, задание CronJob возможно не будет запланировано. Это происходит потому, что CronJob выполняет проверку каждые 10 секунд.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:202]:
```suggestion
Если `startingDeadlineSeconds` имеет значение меньше 10 секунд, задание CronJob, возможно, не будет запланировано. Это происходит потому, что CronJob выполняет проверку каждые 10 секунд.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 19]
```diff
@@ -0,0 +1,233 @@
...
+
+Чтобы подробнее проиллюстрировать эту концепцию, предположим, что задание CronJob настроено на планирование нового задания каждую одну минуту, начиная с `08:30:00`, и его параметр `startingDeadlineSeconds` установлен на 200 секунд. Если контроллер CronJob вдруг не функционирует в течение того же периода, что и в предыдущем примере (с `08:29:00` по `10:21:00`), задание все равно начнется в 10:22:00. Это происходит потому, что контроллер в данном случае проверяет, сколько пропущенных запланированных заданий было за последние 200 секунд (т. е. 3 пропущенных расписания), а не с момента последнего запланированного времени до настоящего момента.
+
+CronJob отвечает только за создание заданий (Jobs), соответствующих его расписанию, а задание, в свою очередь, отвечает за управление подами, которые оно представляет.
+
+## {{% heading "whatsnext" %}}
+
+* Ознакомьтесь с [Подами](/docs/concepts/workloads/pods/) и
+  [Заданиями](/docs/concepts/workloads/controllers/job/) - двух концепциях, на которые опирается CronJobs.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:226]:
```suggestion
  [Заданиями](/docs/concepts/workloads/controllers/job/) — двумя концепциями, на которые опираются CronJobs.
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 20]
```diff
@@ -0,0 +1,233 @@
...
+* Ознакомьтесь с подробным [форматом](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format)
+  полей CronJob `.spec.schedule`.
+* Для инструкции по созданию и работе с CronJob, а также для примера
+  CronJob манифеста, см. раздел [Выполнение автоматизированных задач с помощью CronJob](/docs/tasks/job/automated-tasks-with-cron-jobs/).
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:230]:
```suggestion
  CronJob-манифеста см. раздел [Выполнение автоматизированных задач с помощью CronJob](/docs/tasks/job/automated-tasks-with-cron-jobs/).
```

File: content/ru/docs/concepts/workloads/controllers/cron-jobs.md [hunk 21]
```diff
@@ -0,0 +1,233 @@
...
+* `CronJob` является частью REST API Kubernetes.
```
Comment (shurup) [inline content/ru/docs/concepts/workloads/controllers/cron-jobs.md:231]:
```suggestion
* `CronJob` является частью Kubernetes REST API.
```

**Review by shurup — CHANGES_REQUESTED** [review]
I think you've done a great job dealing with all those Jobs :-) Here are my suggestions to finalise this text.

**Comment by yungserge** [comment]
@shurup Thanks for the feedback! Committed all the suggestions and squashed the commits.

**Comment by shurup** [comment]
Thank you!
/lgtm
/approve

**Review by shurup — APPROVED** [review]

**Pull request merged**